src/Features/Navigation/CoreMenu.php

```diff
@@ -185,22 +185,22 @@ public static function get_items() {
 					'parent'     => 'tools',
 					'title'      => __( 'System status', 'woocommerce-admin' ),
 					'capability' => 'manage_woocommerce',
-					'id'         => 'system-status',
+					'id'         => 'woocommerce/system-status',
```

Comment:
Heads up that these will most likely be moved based on recommendations in https://github.com/woocommerce/woocommerce-admin/pull/5630#pullrequestreview-533786764